packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -177,12 +179,13 @@ export class AutoScalingGroup extends cdk.Construct implements ec2.IClassicLoadB
         // use delayed evaluation
         const machineImage = props.machineImage.getImage(this);
         const userDataToken = new cdk.Token(() => new cdk.FnBase64((machineImage.os.createUserData(this.userDataLines))));
+        const securityGroupsToken = new cdk.Token(() => this.securityGroups.map((sg) => sg.securityGroupId));
```

Comment:
nit (parens not needed): `(sg) => sg.securityGroupId` => `sg => sg.securityGroupId`